src/api-reference/travel/itinerary-v4/v4.itinerary.md

```diff
@@ -9,7 +9,7 @@ The Itinerary API provides clients and authorized partners access to travel itin
 ## <a name="limitations"></a>Limitations
 
 * This API is only available to clients and partners who have been granted access by SAP Concur. Access to this documentation does not provide access to the API. This API is only available in US and EMEA data centers.
-* Events are only raised on a go-forward basis. If you need access to historical data, see the [retrieve trips endpoint](#retrieve-trips)
+* Events are only raised on a go-forward basis. If you need access to historical data, see the [retrieve trips endpoint](#retrieve-trips).
```

Comment:
Just noticed I didn't update the link name for the element on line 2012 to #retrieve-trips. It's currently #get-trips so this link will not work as expected. Should I push up a fix for this?

Reply:
I'll fix it since I have a pending PR.

Reply:
Added, I'll go ahead and have these changes merged and anything else you come up with you can do a PR for.